backend/evals/scannability_qreader.py

```diff
@@ -0,0 +1,46 @@
+import modal
+
+app = modal.App(name="qart-qreader")
+
+
+def download_model():
+    from qrdet import QRDetector
+
+    QRDetector()  # Specify model size here to download non-default ('s' - small)
+
+
+image = (
+    modal.Image.debian_slim()
```

Comment:
```suggestion
    modal.Image.debian_slim(python_version="3.12")
```

To prevent the version from changing depending on the deployment environment